Twitter/Examples/Requests/CreateFastPostCommandExample.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Swashbuckle.AspNetCore.Filters;
+using System;
+using Twitter.Commands;
+
+namespace Twitter.Examples.Requests
```

Comment:
Please move Folder Examples to some other place. Maybe in Contract?

Reply:
I've fixed it.